index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Porfolio</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+
+        <header>
+
+            <nav>
+        
+                <ul class="navbar">
+                    <li>
+                    <a href="#"> <img src="images/logo.svg" alt="logo"> </a>
+                    </li>
+                    <li>
+                    <a href="#"> <img src="images/menu.svg" alt="menu"> </a>
+                    </li>
+
+                </ul>
+            </nav>
+
+        </header>
```

Comment:
great job :clap:  using semantic elements for your header, and using flex for layout 
Please make sure make sure the list items have `list-style-type: none:`